cedar-policy/src/api.rs

```diff
@@ -1382,6 +1383,18 @@ impl Schema {
         ))
     }
 
+    /// Serialize the `Schema` to a JSON value.
+    pub fn to_json_str(&self) -> Result<String, schema_error::SchemaError> {
+        serde_json::to_string(&self.0)
```

Comment:
Unfortunately this will not give the public JSON schema format. IIRC the `Serialize` impl for `cedar_policy_validator::ValidatorSchema` is only to serialize the schema to send it to the Lean.

This function would need to first convert back to a `cedar_policy_validator::SchemaFragment` and then serialize. We don't have any way to do that conversion atm. Perhaps a better solution would to take an approach similar to the `LosslessPolicy` struct we use when serializing policies.

Reply:
Actually, along these lines, we should also have at least one test for each of the new public methods, just to sanity-check that we're getting the results we expect.  (We can also add the JSON serialization to the roundtrip PBT targets, but that would obviously be a separate PR to `cedar-spec`.)